packages/server-wallet/src/wallet/store.ts

```diff
@@ -46,7 +46,10 @@ import {ObjectiveModel, DBObjective} from '../models/objective';
 import {AppBytecode} from '../models/app-bytecode';
 import {LedgerRequest, LedgerRequestType} from '../models/ledger-request';
 import {shouldValidateTransition, validateTransition} from '../utilities/validate-transition';
-import {logger as defaultLogger} from '../logger';
+import {defaultTestConfig} from '../config';
+import {createLogger} from '../logger';
+
+const defaultLogger = createLogger(defaultTestConfig);
```

Comment:
This smells pretty bad. Why is the default logger using the `defaultTestConfig`?

Reply:
Yeah this should probably be cleaned up so that the `Store` **requires** a logger  (or only tries to log when provided one).

Reply:
https://github.com/statechannels/statechannels/issues/2959